core/store/models/bridge_type.go

```diff
@@ -93,9 +93,9 @@ func (bt *BridgeType) SetID(value string) error {
 // password) and a bridge type (with hashed password, for persisting)
 func NewBridgeType(btr *BridgeTypeRequest) (*BridgeTypeAuthentication,
 	*BridgeType, error) {
-	incomingToken := utils.NewBytes32ID()
-	outgoingToken := utils.NewBytes32ID()
-	salt := utils.NewBytes32ID()
+	incomingToken := utils.NewBytes32Secret()
+	outgoingToken := utils.NewBytes32Secret()
+	salt := utils.NewBytes32Secret()
```

Comment:
Let's use the same salt size here as we do with EIs.

Reply:
done